arangod/IResearch/IResearchView.h

```diff
@@ -235,7 +238,7 @@ class IResearchView final: public arangodb::LogicalView {
   void verifyKnownCollections();
 };
 
-}  // namespace iresearch
```

Comment:
are you using clang-format to reformat the code?

Reply:
I have only been formatting the lines I explicitly modified, as per your request last December